web3/main.py

```diff
@@ -204,9 +204,8 @@ def solidityKeccak(cls, abi_types, values):
         return cls.keccak(hexstr=hex_string)
 
     def isConnected(self):
-        for provider in self.providers:
-            if provider.isConnected():
-                return True
+        if self.provider.isConnected():
```

Comment:
This is a nitpick, but can we just say `return self.provider.isConnected()`? Or is it a pythonic thing to have the explicit `return True`?

Reply:
Not sure if it's more or less "pythonic", but I do like your suggestion better, will update!